third_party/tonic/dart_persistent_value.cc

```diff
@@ -43,12 +43,19 @@ void DartPersistentValue::Clear() {
     return;
   }
 
-  if (Dart_CurrentIsolateGroup()) {
-    Dart_DeletePersistentHandle(value_);
-  } else {
-    DartIsolateScope scope(dart_state->isolate());
-    Dart_DeletePersistentHandle(value_);
+  /// TODO(80155): Remove the handle even if the isolate is shutting down.  This
+  /// may cause memory to stick around until the isolate group is destroyed.
+  /// Currently if DartState::IsShuttingDown == true, this code will crash when
```

Comment:
From the perspective of someone reading this in the future, this should read "Previously" right?

Reply:
I was referring to the code inside of the branch, that will crash.  Reworded it.